src/openms/source/ANALYSIS/DENOVO/CompNovoIdentificationBase.cpp

```diff
@@ -548,11 +548,11 @@ for (set<Size>::const_iterator it = used_pos.begin(); it != used_pos.end(); ++it
     }
 
     spec.clear(false);
-    for (PeakSpectrum::ConstIterator it = copy.begin(); it != copy.end(); ++it)
+    for (const auto& spec_copy : copy)
```

Comment:
this is not a copy of a spectrum but a peak in a spectrum so spec_copy is not an ideal name